apps/next-product-starter/src/pages/buy.tsx

```diff
@@ -33,10 +37,19 @@ const Buy: FunctionComponent<BuyProps> = ({bundles}) => {
         <CommerceBundles>
           {bundles.map((bundle) => {
             return (
-              <CommerceBundleDetails bundle={bundle}>
-                <CommerceBundleImage bundle={bundle} />
+              <CommerceBundleDetails
+                getBundleStyles={getBundleStyles}
```

Comment:
these "getter" functions are specific to the product, but they all need them so we will pass them in as callbacks